DataCollector/StatsdDataCollector.php

```diff
@@ -36,7 +36,7 @@ public function reset()
     /**
      * Kernel event
      *
-     * @param Event $event The received event
+     * @param Symfony\Contracts\EventDispatcher\Event|Event $event The received event
```

Comment:
That is the class you import, did you mean \Symfony\Component\EventDispatcher\Event|Event